torch/utils/data/dataloader.py

```diff
@@ -93,6 +93,9 @@ class DataLoader(Generic[T_co]):
         num_workers (int, optional): how many subprocesses to use for data
             loading. ``0`` means that the data will be loaded in the main process.
             (default: ``0``)
+        prefetch_factor (int, optional): multiply factor to affect how many samples
```

Comment:
Maybe a simpler description of this would be "Number of sample loaded in advance by each worker." ? I'm not sure  "multiply factor" is very clear for the end user here.

Reply:
Updated, thanks for pointing out!